data_structures/heap/max_heap.c

```diff
@@ -1,3 +1,8 @@
+/* 
+ * @limits.h defines sizes of integral types.
+ * Constants with the limits of fundamental integral types are defined using this header.
+*/
+#include <limits.h>
```

Comment:
It should be something like this:
```c++
#include <iostream>   /// for IO operations
#include <assert>     /// for assert
```

Reply:
@Panquesito7 Should I add documentation for all the header files in max_heap.c?

Reply:
For all headers (if possible). Thanks. 🙂